packages/postgres-database/src/simcore_postgres_database/migration/versions/baf0ee1c37dc_service_runs_credit_transaction_indexes.py

```diff
@@ -0,0 +1,62 @@
+"""service runs/credit_transaction indexes
+
+Revision ID: baf0ee1c37dc
+Revises: 1c069f85d5fd
+Create Date: 2024-04-29 15:42:52.926095+00:00
+
+"""
+
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "baf0ee1c37dc"
+down_revision = "1c069f85d5fd"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.drop_index(
+        "ix_resource_tracker_credit_transactions_product_name",
+        table_name="resource_tracker_credit_transactions",
+    )
+    op.drop_index(
+        "ix_resource_tracker_credit_transactions_transaction_cla_7bac",
```

Comment:
how do you know this is the index you have to drop? See the suffix `...cla_7ac`. Where does this suffix come from?

Reply:
The name was created automatically by alembic from `CreditTransactionClassification` I guess it was too long.